katello/katello-backup

```diff
@@ -1,405 +1,10 @@
 #!/usr/bin/env ruby
+require "/usr/share/katello/backup.rb"
 
-require 'optparse'
-require 'fileutils'
-require 'date'
-require 'yaml'
-require 'find'
-require 'json'
-require 'highline/import'
+katello_backup = Utilities::Backup.new("foreman-proxy-content", "foreman proxy")
```

Comment:
Utilities is a bit of a broad namespace, I do think you'd be better off with something Katello or Foreman specific.

Reply:
KatelloUtilities seems like it would be pretty explicit about what it contains. I'll switch to that unless there are objections